detector/t5_sentinel/types.py

```diff
@@ -45,10 +53,10 @@ class TokenizerField(BaseModel):
     truncation: bool
     return_tensors: str
 
-    @validator('return_tensors')
+    @validator("return_tensors")
     def verify_return_tensors(cls, v: str):
-        if v not in ['pt', 'tf', 'np']:
-            raise ValueError(f'Returning tensors with {v} is not supported!')
+        if v not in ["pt", "tf", "np"]:
+            raise ValueError(f"Returning tensors with {v} is not supported!")
```

Comment:
Maybe we can remove `tf` and `np` since all other pipelines we built only support `pt` mode.